source/plugin/Assets/GoogleMobileAds/Platforms/GoogleMobileAdsClientFactory.cs

```diff
@@ -11,12 +11,18 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+// Attention Unity 2019 Managed Stripping Level "Medium" or "High" Required Preserve Artributes.
```

Comment:
this comment is unclear. Also 'attributes' is mispelled. Please provide link to relevant information, and clarify comment.